v0.1/elements/electrodeContact.schema.json

```diff
@@ -0,0 +1,66 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/elements/electrodeContact.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SANDS definition of an electrode contact",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type", 
+    "@id",
+    "identifier",
+    "coordinate"
+    ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/ElectrodeContact"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier.",
+      "pattern": "sands/elements/electrodeContact/([A-Za-z0-9_-]+).json"
+    },
+    "identifier": {
+      "type": "string",
+      "description": "Identifier of the electrode contact within files."
+    },
+    "coordinate": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.schema.json#definitions/COORDINATE",
+      "description": "Central coordinate of the electrode contact."
+    },
+    "definedIn":{
+	  "type": "array",
+	  "minItems": 1,
+	  "items": {
+	    "oneOf": [
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json#definitions/CSCS_FILE"},
+	      {"$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE"}
```

Comment:
What is the point of having the option to point to a CSCS file or the file instance in openMINDS? In theory, the CSCS file should be described as a file instances in openMINDS anyway, and then it should rather point to the instance. 

Or is there a scenario where a CSCS file is not integrated as fiel instances to openMINDS?

Reply:
The *CSCS_FILE* is the metadata entity of a single file as we use it in MINDS (v1.0).  
The *FILE_INSTANCE* is the metadata entity of a single file as we will use it in openMINDS (v3.0).

I wanted to make SANDS for now compatible to both versions to be able to play around with it as soon as possible.  
If we realize that we do not need this compatibility we can remove the reference to the *CSCS_FILE* later.

Reply:
Ah, ok. That makes sense. I didn't think of MINDS v1 at all. Then irgnore this comment (and the other 2 below) :)